src/main/java/teammates/common/util/Const.java

```diff
@@ -704,6 +706,7 @@ private Const() {
         public static final String LOCAL_DATE_TIME = "/localdatetime";
         public static final String SESSION_LINKS_RECOVERY = "/sessionlinksrecovery";
         public static final String NATIONALITIES = "/nationalities";
+        public static final String GENERATE_EMAIL = "/generateemail";
```

Comment:
I'm thinking `/email` might be more appropriate. Worthwhile getting more input on this from senior devs.

Reply:
`email` is better. In REST, you are dealing with Resource not RPC.